### PR TITLE
Add daily-digest tooltip message

### PR DESCRIPTION
**Pull request by dan-searle**
More to do...

### CODE REVIEW

File: templates/partials/top/daily-digest.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{{> n-messaging-client/templates/components/n-alert-banner
+theme='neutral'
+contentLong='<p>Click here for your daily digest.</p>'
```
Comment (jonnywyatt) [inline templates/partials/top/daily-digest.html]: The tooltip is on wider viewports only - on mobile it becomes a messaging bar. So the text needs to make sense in that context

Comment (dan-searle) [reply templates/partials/top/daily-digest.html]: I see, thanks. `contentLong` is for the messaging bar, and `contentShort` is for the tooltip?

Comment (jonnywyatt) [reply templates/partials/top/daily-digest.html]: contentShort is shown up to and including S viewport widths, and contentLong beyond that
CSS is in [n-alert-banner](https://github.com/Financial-Times/n-alert-banner/blob/master/src/scss/alert-banner/_alert-banner-mixins.scss#L42-L73)
So yes if tooltips are enabled, they'll use contentLong

Comment (dan-searle) [reply templates/partials/top/daily-digest.html]: Got it, thanks!

**Comment by dan-searle** [comment]
This is parked for now. We're going to implement temporary solution with [n-myft-ui](https://github.com/Financial-Times/n-myft-ui) for the A-B test, and revisit a better solution later.

**Pull request closed without merging**